evap/static/ts/src/datagrid.ts

```diff
@@ -40,10 +41,20 @@ abstract class DataGrid {
     protected constructor({ storageKey, head, container, searchInput }: DataGridParameters) {
         this.storageKey = storageKey;
         this.sortableHeaders = new Map();
+        this.searchableColumnIndices = [];
```

Comment:
This variable is declared in the abstract DataGrid, but only used in the TableGrid subclass - this means that this variable is unused in the ResultsGrid. I think I am fine with this only working with TableGrid for now, because the results page does not seem to need this (@janno42 ?). However, then we should move this to the TableGrid constructor

Reply:
The results page does not need it (for now).